internal/scheduleringester/schedulerdb.go

```diff
@@ -37,7 +37,10 @@ func (s *SchedulerDb) Store(ctx context.Context, instructions *DbOperationsWithM
 			shouldRetry := armadaerrors.IsNetworkError(err) || armadaerrors.IsRetryablePostgresError(err)
 			return shouldRetry, err
 		}, s.initialBackOff, s.maxBackOff)
-		multierror.Append(result, err)
+		multiAppendErr := multierror.Append(result, err)
+		if multiAppendErr != nil {
```

Comment:
Not sure what we should do here.  If we can't append a multierror, not sure if we should panic or just return that error.  Decided to return an error here.